src/Auth0Client/Auth0Client.cs

```diff
@@ -303,32 +302,31 @@ private static bool ScopeHasOfflineAccess(string scope)
 				.Any(e => e.Equals("offline_access", StringComparison.InvariantCultureIgnoreCase));
 		}
 
-		protected void SetupCurrentUser(IDictionary<string, string> accountProperties)
+		protected async void SetupCurrentUser(IDictionary<string, string> accountProperties)
```

Comment:
' async void'  should not be used, please use 'async Task'  but this will be a breaking change